tests/php/ORM/Utf8/Utf8TestHelper.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace SilverStripe\Tests\ORM\Utf8;
+
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\Dev\TestOnly;
+use SilverStripe\ORM\DB;
+
+class Utf8TestHelper implements TestOnly
+{
+    use Injectable;
```

Comment:
Is there any need for injecting this compared with just `new Utf8TestHelper()` in tests? In general with exception of testing the `Injector` explicitly, most of the test code explicitly avoids using the `Injector`.

Reply:
Have removed Injectable